onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -320,8 +321,7 @@ def aten_align_to(self: TensorType, names: Sequence[str]) -> TensorType:
 def aten_all(self: TTensor) -> BOOL:
     """all(Tensor self) -> Tensor"""
 
-    self_rank = op.Size(op.Shape(self))
-    if self_rank == 0:
+    if common_opset.IsScalar(self):
```

Comment:
nit: name bikeshedding. Should we import as `common_op`, keeping the convention of certain `opset` imported as `op`?

Reply:
I simplified the usage and removed common_opset reference.